### ISSUE TITLE
translations need work, please have menus expanded by default?

### ISSUE THREAD

**Issue by kf6shc** [Zweistein2/AgrarMonitor#16]
translations need work, please have menus expanded by default?

**Comment by kf6shc** [comment]
is this mod still being maintained? is there a discord for support or quick questions?

**Comment by Zweistein2** [comment]
Yes, this mod is still being maintained. You can find our discord here: https://discord.gg/r3SacARMRs.

For translations: 
You can help translating stuff by visting: https://weblate.potionlabs.de/projects/agrarmonitor/
